code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -666,6 +666,16 @@
 
 	return
 
+/mob/living/simple_animal/bot/mulebot/Move(turf/simulated/next)
```

Comment:
This doesn't seem necessary?

`process_bot` handles decreasing bloodiness, `Bump` handles stunning them, and `human/Crossed` handles being run over.